lib/models/system.js

```diff
@@ -81,12 +80,6 @@ SystemModel.prototype.buildPayload = function () {
   metrics.mongoPoolCheckedOutConnections = driverMetrics.checkedOut;
   metrics.mongoPoolCreatedConnections = driverMetrics.created;
 
-  const fiberMetrics = getFiberMetrics();
```

Comment:
Would there be any async hooks/ASL metrics that would be useful instead? Can we track how many hooks or ASL stores are being used in the app?

Reply:
I think we can, will research.